lib/manager/travis/extract.spec.ts

```diff
@@ -2,21 +2,109 @@ import { loadFixture } from '../../../test/util';
 import { extractPackageFile } from './extract';
 
 const invalidYAML = loadFixture('invalid.yml');
+const matrixYAMLwithNodeSyntaxString = loadFixture('matrix_jobs.yml');
+const matrixYAMLwithNodeSyntaxArray = loadFixture('matrix_jobs_array.yml');
+const matrixYAMLwithNodeSyntaxArray2 = loadFixture('matrix_jobs_array2.yml');
+const matrixYAMLwithNodeSyntaxAlias = loadFixture('matrix_alias.yml');
+const invalidMatrixYAML = loadFixture('matrix_invalid.yml');
 
 describe('manager/travis/extract', () => {
   describe('extractPackageFile()', () => {
     it('returns empty if fails to parse', () => {
       const res = extractPackageFile('blahhhhh:foo:@what\n');
       expect(res).toBeNull();
     });
+
     it('returns results', () => {
       const res = extractPackageFile('node_js:\n  - 6\n  - 8\n');
       expect(res).toMatchSnapshot();
       expect(res.deps).toHaveLength(2);
     });
+
     it('should handle invalid YAML', () => {
       const res = extractPackageFile(invalidYAML);
       expect(res).toBeNull();
     });
+
+    it('handles matrix node_js syntax with node_js string', () => {
+      const res = extractPackageFile(matrixYAMLwithNodeSyntaxString);
+      expect(res).toMatchInlineSnapshot(`
```

Comment:
Please use `toEqual` matcher for those simple snapshots instead

Reply:
Hi @viceice I'm sorry, I tried t use `toEqual` instead of `matchInlineSnapshot`, but it seems it is only suitable for very simple cases, for example:

```
expect(res).toEqual(`image: redis`);
expect(res).toEqual(`short string`);
```

However in this case I have an object representation:

```
expect(res).toMatchInlineSnapshot(`
         Object {
          "deps": Array [
            Object {
              "currentValue": "11.10.1",
              "datasource": "github-tags",
              "depName": "node",
              "lookupName": "nodejs/node",
            },
          ],
        }
      `);
```

The `toMatchInlineSnapshot()` captures the output for me and substitutes it automatically. It would be hard to use `toEqual` and capture/substitue it myself. It looks `toEqual` doesn't support the Object representation.

Reply:
See suggestion below, needs prettify.